packages/terra-site/src/examples/collapse/OpenCloseEventCollapse.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Collapse from 'terra-collapse/src/Collapse';
+
+const OpenCloseEventCollapse = () => (
+  <Collapse
+    closedButtonText="On Open and On Close Events"
+    onOpen={() => alert('Fired when collapse is opened')}
```

Comment:
@JakeLaCombe has mentioned on one of my code reviews we should avoid the use of alerts in our examples to consumers. He suggests building a full React component that updates the state with the callback value, and display that in a paragraph underneath. Not sure if we want to keep this structure.